src/main/java/org/learn_java/bot/data/repositories/BanRepository.java

```diff
@@ -0,0 +1,11 @@
+package org.learn_java.bot.data.repositories;
+
+import java.util.List;
+import org.learn_java.bot.data.entities.Ban;
+import org.springframework.data.jpa.repository.JpaRepository;
+// import org.springframework.stereotype.Repository;
+
+// @Repository
+public interface BanRepository extends JpaRepository<Ban, Long> {
+  List<Ban> findByUser_ID(long user_id);
```

Comment:
Rename this once you fix the field name as well

Reply:
Rename to ? haven't had the opportunity to work with databases so